www/js/app.js

```diff
@@ -98,6 +98,15 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
+ 
+    var darkPreference = window.matchMedia("(prefers-color-scheme : dark");
+    darkPreference.addEventListener("change", e=> {
```

Comment:
Don't use arrow functions, as they are not compatible with all the JavaScript versions we support.